### DISCUSSION TITLE
How to allow a role to insert only users with another specific role?

### DISCUSSION THREAD

**Discussion by wemersonrv** [hasura/graphql-engine#7218] [Question]
Hello, i'm setting permissions for my users table, and need to allow role `gestor` to only add new users with the value `sales` defined for the field role, and do not accept another rules. So i did as below:
![image](https://user-images.githubusercontent.com/2028673/125174477-2bbebb80-e19c-11eb-8254-bf6621c107ff.png)

But don't works. It still allow insert users with another roles. See below.:
![image](https://user-images.githubusercontent.com/2028673/125174626-17c78980-e19d-11eb-8a69-2db1492808b3.png)

How can i do that properly?

**Comment by leoalves** [comment]
You can do that setting the column preset
![image](https://user-images.githubusercontent.com/3867140/125512651-a495e0a2-1aee-41e2-80bb-44624aea8729.png)
